tests/manual/lxd-ceph-cmr.yaml

```diff
@@ -0,0 +1,55 @@
+default-base: ubuntu@22.04/stable
+
+saas:
+  graf:
+    url: microk8s:admin/cos.grafana-dashboards
+  prom:
+    url: microk8s:admin/cos.prometheus-receive-remote-write
```

Comment:
Including this section makes the bundle slightly less versatile:
- You must have a cos model and must name the offers exactly this way.
- You must deploy this bundle only after deploy cos.

Wdyt?